src/tests/end-to-end/tests/details-view/overview.test.ts

```diff
@@ -15,6 +15,7 @@ describe('Details View -> Overview Page', () => {
     beforeAll(async () => {
         browser = await launchBrowser({ suppressFirstTimeDialog: true });
         targetPage = await browser.newTargetPage();
+        await browser.newPopupPage(targetPage); // Required for the details view to register as having permissions/being open
```

Comment:
same here: Did I forget to delete this after changing how we launch the browser (i.e. after adding the `addLocalhostToPermissions` flag)

Reply:
same as other one